Models/WebResource.cs

```diff
@@ -30,8 +30,9 @@ public string Name
                 {
                     var parts = value.Split('_');
                     CustomizationPrefix = parts[0];
+                    //RemoteName = value.Replace($"{parts[0]}_", string.Empty);
                     string pattern = $"^{parts[0]}_";
-                    RemoteName = Regex.Replace(value, pattern, string.Empty);
+                    RemoteName = Regex.Replace(value, pattern, String.Empty);
```

Comment:
Done - tried fixing it online but didn't take. I also just added font file type handling -- let me know if anything there needs to be tweaked too.